charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -333,6 +374,37 @@ data:
         guest:
           dangerouslyAllowOutsideDevelopment: true
   {{- end }}
+  {{- if .Values.rhsso.enabled }}
+  app-config-auth.oidc.yaml: |
+    signInPage: oidc
+    auth:
+      session:
+        secret: {{ randAlphaNum 32 }}
+      providers:
+        oidc:
+          development:
+            metadataUrl: {{ $OIDC_METADATA_URL }}
+            clientId: {{ $.Values.rhsso.client.username }}
```

Comment:
Storing credentials in a template is a security risk as it is visible in the CR's status. Is there a workaround to this?